components/tools/ToolsCard.tsx

```diff
@@ -91,17 +85,22 @@ export default function ToolsCard({ toolData }: ToolsCardProp) {
           <div className='relative'>
             <Paragraph typeStyle={ParagraphTypeStyle.sm}>
               <span
-                ref={descriptionRef}
-                className={`w-full ${showMoreDescription ? 'cursor-pointer' : ''}`}
+                className={`w-full ${isTruncated ? 'cursor-pointer' : ''}`}
                 onMouseEnter={() =>
                   setTimeout(() => {
-                    if (showMoreDescription) setShowDescription(true);
+                    if (isTruncated) setShowDescription(true);
                   }, 500)
                 }
               >
-                <TextTruncate element='span' line={3} text={toolData.description} />
+                <div
+                  ref={descriptionRef}
+                  className={`line-clamp-3 max-h-[4.5em] overflow-hidden text-ellipsis ${isTruncated && 'after:content-["..."]'}`}
+                >
+                  {toolData.description}
```

Comment:
How are we truncating the text here?

Reply:
The text is truncated using `line-clamp-3`, which limits the text to three lines. If isTruncated is true, the `after:content-["..."] `adds ellipsis (...) after the truncated text.

The styles `max-h-[4.5em] overflow-hidden text-ellipsis` are not relevant. I have removed them.